ENDPOINT_STATUS.md

```diff
@@ -1,36 +1,39 @@
-- BILLING TEST NOT OK = test bill is not working
-- BILLING OK = billing has been attempted and verified to work
-- BILLING NOT VERIFIED = billing probably works, but it has not been verified with a real bill yet
-- BILLING NOT OK = billing does not work
-
+| Test Result          | Explanation                                                                                           |
+| -------------------- | ----------------------------------------------------------------------------------------------------- |
+| CODE TEST OK         | test codes are working                                                                                |
+| CODE TEST NOT OK     | test codes are not working                                                                            |
+| CODE OK              | real code validation has been attempted and verified to work                                          |
+| CODE NOT VERIFIED    | real code validation probably works, but it has not been tested on a real code yet                    |
+| CODE NOT OK          | real code validation does not work                                                                    |
+| BILLING TEST OK      | test bill is working (the APIs dont support this completely, but as far as you can get it is working) |
+| BILLING TEST NOT OK  | test bill is not working                                                                              |
+| BILLING OK           | billing has been attempted and verified to work                                                       |
+| BILLING NOT VERIFIED | billing probably works, but it has not been verified with a real bill yet                             |
+| BILLING NOT OK       | billing does not work                                                                                 |
 
 ## Endpoints
 
-- diga.bitmarck-daten.de - 85 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- diga-api.tk.de/diga/api/public/rest - 1 company - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- da-api.gkvi.de - 5 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- diga.kkh.de - 1 company - CODE TEST OK - CODE OK
-- itscare.da-api.aok.de - 3 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING NOT VERIFIED
-- kubus-it.da-api.aok.de - 2 companies - CODE TEST OK - CODE NOT VERIFIED - BILLING TEST OK - BILLING NOT VERIFIED
-- arge.da-api.aok.de - 3 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- diga.apimisc.de - 3 companies - CODE TEST OK - CODE OK - BILLING TEST NOT OK - BILLING OK
+| Api                                 | #Companies | CODE TEST | CODE         | BILLING TEST | BILLING      |
+| ----------------------------------- | ---------- | --------- | ------------ | ------------ | ------------ |
+| diga.bitmarck-daten.de              | 86         | OK        | OK           | OK           | OK           |
```

Comment:
This has been updated from 85 -> 86 as a new mapping file was published. I did not check which insurance was added but only which api provider is used. Fortunately, it is no new provider :)